gems/aws-sdk-core/lib/aws-sdk-sso.rb

```diff
@@ -7,8 +7,6 @@
 #
 # WARNING ABOUT GENERATED CODE
 
-require 'aws-sdk-core'
```

Comment:
I think this could cause issues in some cases - if you only require aws-sdk-sso, you'd be missing core